timeless/__init__.py

```diff
@@ -96,13 +96,32 @@ def register_endpoints(app):
         "/api/comments/",
         pk="comment_id"
     )
-    register_api(
-        app,
-        reservations_views.SettingsView,
-        "settings.api",
-        "/reservations/settings/"
-    )
     app.register_blueprint(floors_views.bp)
     app.register_blueprint(table_shapes_views.bp)
     app.register_blueprint(reservations_views.bp)
+    app.add_url_rule(
+        "/reservations/settings/",
+        view_func=reservations_views.SettingsListView.as_view(
+            "reservation_settings_list")
```

Comment:
@vladarefiev let's leave a puzzle to make a function for registering template views